docs/DEPENDENCIES.md

```diff
@@ -1,3 +1,107 @@
 ## Dependencies
 
-`Abstruse` requires `SQLite3` and `Docker` to be installed.
+`Abstruse` requires `sqlite3` and `Docker` to be installed.
+
+### Contents
+
+* [Ubuntu Linux](#ubuntu-linux)
+* [MacOS](#macos)
+* [Windows](#windows)
+
+
+### Ubuntu Linux
+
+1. Install Node.JS
+
+There are many ways installing Node.JS on Ubuntu. We will use approach where you can later update, uninstall or use another version of Node.JS easily.
```

Comment:
I think that 'We will use an approach' or 'the approach' would be better here...

Reply:
updated.